gssapi/raw/ext_ggf.pyx

```diff
@@ -0,0 +1,47 @@
+GSSAPI="BASE"
+
+from gssapi.raw.cython_types cimport *
+from gssapi.raw.ext_buffer_sets cimport *
+from gssapi.raw.cython_converters cimport c_get_mech_oid_set
+from gssapi.raw.misc import GSSError
+from gssapi.raw.oids cimport OID
+from gssapi.raw.sec_contexts cimport SecurityContext
+
+cdef extern from "python_gssapi_ext.h":
+    OM_uint32 gss_inquire_sec_context_by_oid(OM_uint32 *min_stat,
+                                  const gss_ctx_id_t context_handle,
+                                  const gss_OID desired_object,
+                                  gss_buffer_set_t *data_set) nogil
+
+
+def inquire_sec_context_by_oid(SecurityContext context not None, OID mech not None):
+    """
+    Args:
```

Comment:
This needs to follow the proper documentation format, including listing what it does, and what it returns.  In general, `python-gssapi` documentation should be more-or-less usable without having to reference the main RFC.